connector/src/main/java/org/geysermc/connector/entity/living/animal/horse/AbstractHorseEntity.java

```diff
@@ -44,6 +45,10 @@ public AbstractHorseEntity(long entityId, long geyserId, EntityType entityType,
 
         // Specifies the size of the entity's inventory. Required to place slots in the entity.
         metadata.put(EntityData.CONTAINER_BASE_SIZE, 2);
+        // Add dummy health attribute since LivingEntity updates the attribute for us
+        attributes.put(AttributeType.HEALTH, AttributeType.HEALTH.getAttribute(20, 20));
+        // Add horse jump strength attribute to allow donkeys and mules to jump
+        attributes.put(AttributeType.HORSE_JUMP_STRENGTH, AttributeType.HORSE_JUMP_STRENGTH.getAttribute(0.5f, 2));
```

Comment:
Does this cause an issue with llamas?

Reply:
I disabled power jumping on llamas in the latest commit.